test/zdtm/lib/ns.c

```diff
@@ -52,6 +53,17 @@ static int prepare_mntns(void)
 		return -1;
 	}
 
+	/*
+	 * The mnt_ro_root test makes "/" mount readonly, but we still
+	 * want to write logs to /zdtm/static/ so let's make it separate
+	 * writable mount.
+	 */
+	snprintf(testdir, sizeof(testdir),  "%s/zdtm/static/", root);
+	if (mount(testdir, testdir, NULL, MS_BIND, NULL)) {
```

Comment:
```
sudo ./zdtm.py run -t zdtm/static/mnt_ext_auto -f uns
userns is supported
Checking feature mnt_id
mnt_id is supported
=== Run 1/1 ================ zdtm/static/mnt_ext_auto
===================== Run zdtm/static/mnt_ext_auto in uns ======================
Start test
./mnt_ext_auto --pidfile=mnt_ext_auto.pid --outfile=mnt_ext_auto.out --dirname=mnt_ext_auto.test
Can't bind-mount testdir: Invalid argument
The test failed (0, 1)
make: *** [Makefile:519: mnt_ext_auto.pid] Error 1
 Test zdtm/static/mnt_ext_auto FAIL at ['make', '--no-print-directory', '-C', 'zdtm/static', 'mnt_ext_auto.pid'] 
##################################### FAIL #####################################
```

Reply:
Thanks for pointing this out! I think adding this bindmount for all tests was a mistake. Let's add it to `mnt_ro_root` test only, thus we will know for sure that we didn't break other tests.